snuba/cli/bootstrap.py

```diff
@@ -100,11 +101,25 @@ def bootstrap(
     # tables or distributed tables, etc.
 
     # Create the tables for every dataset.
+    clickhouse_tables = ClickhousePool().execute("show tables")
+    existing_tables = [row[0] for row in clickhouse_tables]
     for name in DATASET_NAMES:
         dataset = get_dataset(name)
 
         logger.debug("Creating tables for dataset %s", name)
+        run_migrations = False
         for statement in dataset.get_dataset_schemas().get_create_statements():
-            logger.debug("Executing:\n%s", statement)
-            ClickhousePool().execute(statement)
+            if statement.table_name not in existing_tables:
+                # This hack is needed since ClickHouse would try to execute the
+                # SELECT statement that defines a Materialized View even if the View
+                # exists. This may require the migrate script to run instead to make
+                # the SELECT statement work.
```

Comment:
This comment is confusing to me. Isn't the statement a `CREATE` statement, not a `SELECT`?

I also don't really understand what this means/how it relates to what is happening here:

> This may require the migrate script to run instead to make the SELECT statement work.

Is there a different/clearer wording?

Reply:
Will try to rewrite. And yes the clickhouse behavior there is awfully counterintuitive.